cmd/jujud/agent/caasoperator_test.go

```diff
@@ -0,0 +1,228 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package agent
+
+import (
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/juju/cmd"
+	"github.com/juju/cmd/cmdtesting"
+	jc "github.com/juju/testing/checkers"
+	gc "gopkg.in/check.v1"
+	"gopkg.in/natefinch/lumberjack.v2"
+
+	"github.com/juju/juju/agent"
+	"github.com/juju/juju/cmd/jujud/agent/agenttest"
+	"github.com/juju/juju/state"
+	coretesting "github.com/juju/juju/testing"
+	"github.com/juju/juju/tools"
+	"github.com/juju/juju/worker/caasoperator/commands"
+	"github.com/juju/juju/worker/logsender"
+)
+
+type CAASOperatorSuite struct {
+	coretesting.GitSuite
+	agenttest.AgentSuite
+}
+
+var _ = gc.Suite(&CAASOperatorSuite{})
+
+func (s *CAASOperatorSuite) SetUpSuite(c *gc.C) {
+	s.GitSuite.SetUpSuite(c)
```

Comment:
Is this GitSuite needed? I'm guessing you cribbed this from unit_test.go. I don't think the CAAS operator or tests should need it.

Reply:
yeah, not needed